jib-core/src/main/java/com/google/cloud/tools/jib/image/ReproducibleLayerBuilder.java

```diff
@@ -145,9 +152,13 @@ public Blob build() {
     for (FileEntry layerEntry : layerEntries) {
       // Adds the entries to uniqueTarArchiveEntries, which makes sure all entries are unique and
       // adds parent directories for each extraction path.
-      TarArchiveEntry entry =
-          new TarArchiveEntry(
-              layerEntry.getSourceFile().toFile(), layerEntry.getExtractionPath().toString());
+      TarArchiveEntry entry;
+      try {
+        entry = new TarArchiveEntry(
+                layerEntry.getSourceFile(), layerEntry.getExtractionPath().toString());
+      } catch (IOException e) {
+        throw new UncheckedIOException(e);
```

Comment:
Likewise, have `build() throws IOException`. All will be fine; no more propagated changes beyond this point.

Reply:
done